src/templates/fr/2019/methodology.html

```diff
@@ -163,6 +163,10 @@ <h2 id="dataset">À propos du jeu de données</h2>
 
         <p>
         Par exemple, pour obtenir le nombre médian d’octets de JavaScript servis sur les pages pour ordinateurs de bureau et périphériques mobiles, regardez <a href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/master/sql/2019/01_JavaScript/01_01b.sql">01_01b.sql</a>:
+        
+
+        <p>
+          Par exemple, pour comprendre le nombre médian d'octets de JavaScript sur les sites pour ordinateurs de bureau et périphériques mobiles, voir <a href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/master/sql/2019/01_JavaScript/01_01b.sql">01_01b.sql</a>:
```

Comment:
You seem to have repeated the previous paragraph?

This was the line to translate above that paragraph:

```
        <p>
          <aside>Please note that some of these queries are quite large and can be <a href="https://cloud.google.com/bigquery/pricing">expensive</a> to run yourself, as BigQuery is billed by the terabyte. For help controlling your spending, refer to Tim Kadlec's post <a href="https://timkadlec.com/remembers/2019-12-10-using-bigquery-without-breaking-the-bank/">Using BigQuery Without Breaking the Bank</a>.</aside>
        </p>
```

Reply:
Though nice to know your translation is semi-consistent 😊

Though a couple of words different so pick which one you like and change in this same PR if you want.